### PR TITLE
update take action page

### PR DESCRIPTION
**Pull request by jamesrhaley**
update to the take action page, plus a slight tweak to the panel on the home page.  I made the panel very light.  Also I adjusted the font size of the share links on the video page.

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -369,23 +369,58 @@ pre {
...
+    @include button-colors($color-facebook);
+  }
+  &.reddit {
+    @include button-colors($color-reddit);
+  }
+  &.twitter {
+    @include button-colors($color-twitter);
+  }
+  &.youtube {
+    @include button-colors($color-youtube);
```
Comment (Rio517) [inline _sass/_base.scss:404]: I know there are some existing buttons for FB, TW, etc.  Just wanted to make sure you saw those and determined they couldn't be reused.

File: _sass/_base.scss [hunk 2]
```diff
@@ -358,10 +358,10 @@ pre {
   font-weight: 400;
 
   .fa {
-    float: left;
+    // float: left;
     font-size: 1.5em;
-    position: absolute;
-  }
+    padding-right:5px;
```
Comment (Rio517) [inline _sass/_base.scss:363]: It was positoned absolute and floated left - odd.  Good flag.  These new updates looked ok gobally.  You can just delete the old css - no need to comment out.

File: _includes/panels/panel_action_w_call_congress.html [hunk 3]
```diff
@@ -9,10 +9,10 @@
       <p>Join the Mayday team of Citizen Lobbyists fighting together for fundamental reform.</p>
       <form class="join" id="js-form-call-congress">
         <div class="row">
-          <div class="col-md-8">
+          <div class="col-md-8 form-left-sm">
```
Comment (Rio517) [inline _includes/panels/panel_action_w_call_congress.html:12]: Where possible, avoid adding new classes.  I'm guessing you explored that, but just try to keep minimizing where possible.  Thanks!

Comment (jamesrhaley) [inline _includes/panels/panel_action_w_call_congress.html:12]: my biggest goal was to not change the dom.  Basically the forms have different varieties of margins that need to all be the same but I don't want to affect any code that you have written by accident.

File: _sass/_base.scss (hunk 2 above)
Comment (jamesrhaley) [inline _sass/_base.scss:363]: sure will do

**Pull request merged**